File: dotnetapi/Models/Drug.cs
namespace Business.Entities
{
    public class Drug
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int? ManufacturerId { get; set; }
        public string Milligrams { get; set; }
        public decimal? Cost { get; set; }
    }
}

File: backend/src/main/java/com/kevinlemein/backend/model/Drug.java
package com.kevinlemein.backend.model;

import jakarta.persistence.*;
import lombok.*;

import java.math.BigDecimal;

@Entity
@Table(name = "drugs")
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Drug {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false)
    private String name;

    @Column(name = "manufacturer_id")
    private Long manufacturerId;

    private Integer milligrams;

    private String category;

    @Column(precision = 10, scale = 2)
    private BigDecimal cost;

    // Current stock on hand. Decremented transactionally whenever a
    // prescription is created for this drug -- see PrescriptionService.
    @Column(nullable = false)
    private Integer quantity;
}